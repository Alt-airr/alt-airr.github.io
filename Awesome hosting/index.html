<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Awesome Hosting</title>
    <link rel="stylesheet" href="css/style.css">
    <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700,800,900&amp;subset=cyrillic" rel="stylesheet">
</head>
<body>
  
  
   <header>
       <div class="container">
          <div class="top clearfix">
           <img class="logo" src="img/Logo.png" alt="logo">
           
           
           <div class="comein">
           <p><span>Support</span> : info@webhosting.com
            <a href="">Register</a> 
            <a href="">Login</a> 
           </p>  
            </div>    
            </div>
              <div class="menu">
                <ul>
                    <li><a href="#">Home</a></li>
                    <li><a href="#">About Us</a></li>
                    <li><a href="#">Services</a></li>
                    <li><a href="#">Hosting plans</a></li>
                    <li><a href="#">Domain</a></li>
                    <li><a href="#">FAQ</a></li>
                    <li><a href="#">Testimonials</a></li>
                    <li><a href="#">Blog</a></li>
                    <li><a href="#">Support</a></li>
                    <li><a href="#">Contact Us</a></li>
                </ul>
               </div>
       </div>
   </header> 
   
   <section id="second_section">
       <div class="container">
           
          <h1>Awesome hosting</h1>
           <p>with unlimited features</p>
       </div>
   </section>
   
   
   <section id="third_section">
       <div class="container">
          
          <div class="glass_area">
           <img class="glass" src="img/glass.png" alt="glass">
           
           <p class="search_your">search your <span>domain</span></p>
           </div>
           
           <form action="search" class="search_bar">
           <p class="search"><input class="search_line" type="search" name="q" placeholder="Enter your domain name">
           
           <select name="year" id="year">
               <option value="">0-1 year</option>
               <option value="">1-3 years</option>
               <option value="">3-7 years</option>
               <option value="">7-10 years</option>
               <option value="">>10 years</option>
           </select>
           
           <select name="domain_type" id="domain_type">
               <option value="">.com</option>
               <option value="">.org</option>
               <option value="">.ua</option>
               <option value="">.uk</option>
           </select>
           
           <button type="submit">
               Search
           </button>
           </form>
       </div>
   </section>
   
   
   <section>
       <div class="container">
          
          
           <div class="benefits_area clearfix">
         <img src="img/Shape%204.png" alt="">   
           <p>Our <span>benefits</span></p>
           </div>
           
           
           <div class="benefits clearfix">
              <div class="benefit">
                  <img src="img/img1.png" alt="">
                  <p><span>99,9%</span><br>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been. </p>
              </div>
              
              
               <div class="benefit">
                  <img src="img/img2.png" alt="">
                  <p><span>99,9%</span><br>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been. </p>
              </div>
              
              <div class="benefit">
                  <img src="img/img3.png" alt="">
                  <p><span>99,9%</span><br>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been. </p>
              </div>
             
              <div class="benefit suka">
                  <img src="img/img4.png" alt="">
                  <p><span>99,9%</span><br>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been. </p>
              </div>
              <div class="benefit">
                  <img src="img/img5.png" alt="">
                  <p><span>99,9%</span><br>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been. </p>
              </div>
              <div class="benefit">
                  <img src="img/img6.png" alt="">
                  <p><span>99,9%</span><br>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been. </p>
              </div>
           </div>
           
           
           
       </div>
   </section>
   
   
   

   
   <section class="fiveth_section">
       <div class="container">
          
          <div class="left">
             <div class="hat">
              <img src="img/Shape%204.png" alt="">
              <p>Now <br><span>What is hot</span></p>
              </div>
              <img src="img/offer.png" alt="">
              
              
              
          </div>
          
        
          
          <div class="right">
              <div class="hat">
              <img src="img/Shape%204.png" alt="">
              <p>Our <br><span>features</span></p>
              </div>
              
              <ul>
                  <li>Lorem Ipsum is simply dummy. </li>
                  <li>Discovered the undoubtable. </li>
                  <li>Various versions have evolved. </li>
                  <li>Suffered alteration in some. </li>
                  <li>Lorem ipsum dolor sit </li>
                  <li>Lorem ipsum dolor sit </li>
                  <li>Lorem ipsum dolor sit </li>
                  <li>Lorem ipsum dolor sit </li>
                  
              </ul>
              
          </div>
          
          <div class="lebutton">
             <div class="butt">
              <button class="buttView">
               View All
               </button>
              </div>
          </div>
          
           
       </div>
   </section>
   
   
   
   
   <section id="sixth_section">
       <div class="container">
           
          <div class="hat">
              <img src="img/Shape%204.png" alt="">
              <p>Our <br><span>hosting</span></p>
          </div>
           
           
         <div class="tariffs">

             <div class="tariff">
                 <p class="price">
                     <span>$ 5</span> <br> per month

                 </p>
                 <div class="plan">
                     Standart plan
                 </div>

                 <ul class="character">
                     <li>10GB Disk Space</li>
                     <li>1TB Bandwidth</li>
                     <li>Free DDoS Protection</li>
                     <li>Free Daily Backups</li>
                     <li>Managed Hosting</li>
                 </ul>


                 <div class="plan_bottom">
                     <button class="signup" type="submit">Signup</button>
                 </div>
             </div>
             
             
             <div class="tariff">
                 <div class="tariff">
                 <p class="price">
                     <span>$ 6</span> <br> per month

                 </p>
                 <div class="plan">
                     Advenced plan
                 </div>

                 <ul class="character">
                     <li>10GB Disk Space</li>
                     <li>1TB Bandwidth</li>
                     <li>Free DDoS Protection</li>
                     <li>Free Daily Backups</li>
                     <li>Managed Hosting</li>
                 </ul>


                 <div class="plan_bottom">
                     <button class="signup" type="submit">Signup</button>
                 </div>
             </div>
             </div>
             
             
             <div class="tariff">
                <div class="tariff">
                 <p class="price">
                     <span>$ 8</span> <br> per month

                 </p>
                 <div class="plan">
                     Buisness plan
                 </div>

                 <ul class="character">
                     <li>10GB Disk Space</li>
                     <li>1TB Bandwidth</li>
                     <li>Free DDoS Protection</li>
                     <li>Free Daily Backups</li>
                     <li>Managed Hosting</li>
                 </ul>


                 <div class="plan_bottom">
                     <button class="signup" type="submit">Signup</button>
                 </div>
             </div> 
             </div>
             
             
             <div class="tariff">
                 <div class="tariff">
                 <p class="price">
                     <span>$ 9</span> <br> per month

                 </p>
                 <div class="plan">
                     Gold plan
                 </div>

                 <ul class="character">
                     <li>10GB Disk Space</li>
                     <li>1TB Bandwidth</li>
                     <li>Free DDoS Protection</li>
                     <li>Free Daily Backups</li>
                     <li>Managed Hosting</li>
                 </ul>


                 <div class="plan_bottom">
                     <button class="signup" type="submit">Signup</button>
                 </div>
             </div>
             </div>
         </div>  
           
           
           
           
           
       </div>
   </section>
   
   
   
   
   
   
   
   
   
   
   <section class="seventh_secction">
       <div class="container">
           
             <div class="hat newsletter">
           <img class="glass" src="img/letter.png" alt="glass"> <br>
           
           <p class="search_your">Sign up your <span>Newsletter</span></p>
           </div>
           
           <form class="letters_search" action="search" >
           <p><input class="letter_search" type="search" name="q" placeholder="Enter your domain name">
           
         
           <button type="submit">
               Sign up now
           </button>
           </p>
           </form>
       </div>
           
           
           
       
   </section>
   
   
   
   <footer>
       <div class="container">
          
          
           <div class="footer_logo">
               <img src="img/logo2.png" alt="logo"> 
               <div class="blet">Hosting</div><br>
               <p class="rights">2015 © Allright Reserved</p>
           </div>
           
           
           <div class="footer_quickLinks">
             <h3>Quick links</h3>
              <a href="#">Home</a>
              <a href="#">About Us</a>
              <a href="#">Service</a>
              <a href="#">Hosting plans</a>
              <a href="#">Domains</a> 

           </div>
           
            <div class="footer_Links">
                <a href="#">Faq</a>
              <a href="#">Blog</a>
              <a href="#">Support</a>
              <a href="#">Contact</a>
            
           </div>
           
           
           <div class="footer_contactUs">
              <h3>Contact us</h3>
               <p>
                   <span>Address :</span> 9867 lipsum text here
There are many variations <br>
<span>Email :</span>  info@hosting.com <br>
<span>Call :</span>  +87 7765 557 888
               </p>
               
           </div>
           <div class="footer_socialMedia">
               <h3>Social media</h3>
               <img src="img/sc.png" alt="">
           </div>
           
       </div>
   </footer>
   
   
  
</body>
</html>








