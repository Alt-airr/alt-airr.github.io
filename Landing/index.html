<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <title>Landing</title>

    <link rel="stylesheet" href="css/main.css">
    <link rel="stylesheet" href="css/animate.css">
   <link href="https://fonts.googleapis.com/css?family=Open+Sans&display=swap" rel="stylesheet">
    
</head>
<body>

    <section class="universe" id="universe">
            <h2 class="h2 f-left hide" id="universe__h2">Our Universe . . .</h2>
            <p class="f-left p hide" id="universe__p">
                ...is all of space and time and their contents, including planets, stars, galaxies, and all other forms of matter and energy. While the spatial size of the entire Universe is unknown, it is possible to measure the size of the observable universe, which is currently estimated to be 93 billion light years in diameter...
            </p>
            <img src="css/img/universe2.png" alt="" class="img hide" id="universe__img1">
    </section>


    <section class="milky" id="milky">
            <img class="wow fadeIn" id="milky__img1" 
            data-wow-duration='4s' 
            data-wow-delay='7s' src="css/img/mw2.png" alt="" align="left">

            <h2 class="h2 wow fadeIn" 
            data-wow-duration='3s' 
            data-wow-delay='3s'>Milky Way</h2>
            
            <p class="p wow fadeIn" data-wow-duration='3s' data-wow-delay='4.5s'>
                ...is the galaxy that contains the Solar System, with the name describing the galaxy's appearance from Earth: a hazy band of light seen in the night sky formed from stars that cannot be individually distinguished by the naked eye...
            </p>
    </section>


    <section class="solar" id="solar">
            <img id="mer" class="wow fadeIn" 
            data-wow-duration='3s' 
            data-wow-delay='5s' 
            data-wow-offset='300' 
            src="css/img/system/mercury.png" alt="">
            
            <img id="ven" class="wow fadeIn" 
            data-wow-duration='3s' 
            data-wow-delay='5.3s' 
            data-wow-offset='300' 
            src="css/img/system/venus.png" alt="">
            
            <img id="ea" class="wow fadeIn" 
            data-wow-duration='3s' 
            data-wow-delay='5.6s' 
            data-wow-offset='300' 
            src="css/img/system/earth.png" alt="">
            
            <img id="ma" class="wow fadeIn" 
            data-wow-duration='3s' 
            data-wow-delay='6s' 
            data-wow-offset='300' 
            src="css/img/system/mars.png" alt="">
            
            <img id="ju" class="wow fadeIn" 
            data-wow-duration='3s' 
            data-wow-delay='6.3s' 
            data-wow-offset='300' 
            src="css/img/system/jupiter.png" alt="">
            
            <img id="sa" class="wow fadeIn" 
            data-wow-duration='3s' 
            data-wow-delay='6.6s' 
            data-wow-offset='300' 
            src="css/img/system/saturn.png" alt="">
            
            <img id="ur" class="wow fadeIn" 
            data-wow-duration='3s' 
            data-wow-delay='6.9s' 
            data-wow-offset='300' 
            src="css/img/system/uran.png" alt="">
            
            <img id="ne" class="wow fadeIn" 
            data-wow-duration='3s' 
            data-wow-delay='7.2s' 
            data-wow-offset='100' 
            src="css/img/system/neptune.png" alt="">
            
            <img id="pl" class="wow fadeIn" 
            data-wow-duration='3s' 
            data-wow-delay='7.5s' 
            src="css/img/system/pluton.png" alt="">

            <h2 class="h2 wow fadeIn" 
            data-wow-duration='7s' 
            data-wow-delay='1s'>Solar System</h2>

            <p class="p wow fadeIn" 
            data-wow-duration='7s' 
            data-wow-delay='3s'>
                ...is the gravitationally bound planetary system of the Sun and the objects that orbit it, either directly or indirectly Of the objects that orbit the Sun directly, the largest are the eight planets, with the remainder being smaller objects, such as the five dwarf planets and small Solar System bodies...
            </p> 
    </section>


    <section class="earth" id="earth">
            <p id="earth__p1" class="p wow fadeIn" 
               data-wow-duration='7s' 
               data-wow-delay='3s'>
                Earth is the third planet from the Sun and the only astronomical object known to harbor life. According to radiometric dating and other sources of evidence, Earth formed over 4.5 billion years ago. Earth's gravity interacts with other objects in space, especially the Sun and the Moon, Earth's only natural satellite...
            </p>
            <p id="earth__p2" class="p wow fadeIn" 
               data-wow-duration='5s' 
               data-wow-delay='5s'>
                Earth's axis of rotation is tilted with respect to its orbital plane, producing seasons on Earth. The gravitational interaction between Earth and the Moon causes tides, stabilizes Earth's orientation on its axis and gradually slows its rotation.Earth is the densest planet in the Solar System and the largest and most massive of the four terrestrial planets...
            </p>
    </section>

<section id="landing">
</section>

<section id="landing2">
</section>

<embed src="css/bg-music.wav" autostart="true" hidden="true" loop="false"> </embed>

    <script src="js/wow.min.js"></script>
    <script> new WOW().init();</script>
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.0/jquery.min.js">
    </script>
    <script src="js/Script.js"> </script>
</body>

</html>