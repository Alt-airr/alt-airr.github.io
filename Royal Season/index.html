<!DOCTYPE html>
<html lang="ru">
<head>
    <meta charset="UTF-8">
    <title>Hotel</title>
    <link rel="stylesheet" href="css/style.css">
     <link rel="stylesheet" href="css/fonts.css">
     <link href="https://fonts.googleapis.com/css?family=Niconne&display=swap&subset=latin-ext" rel="stylesheet">
     <link href="https://fonts.googleapis.com/css?family=Monsieur+La+Doulaise&display=swap" rel="stylesheet">
    <link href="https://fonts.googleapis.com/css?family=Miss+Fajardose&display=swap" rel="stylesheet">
</head>
<body>
    
    <header>
       <div class="container">
           
           
           
           
           
            <div class="menu">
                <ul>
                    <li><a href="#">Home</a></li>
                    <li><a href="#">About Us</a></li>
                    <li><a href="#">Services</a></li>
                    <li><a href="#">Hosting plans</a></li>
                    <li><a href="#">FAQ</a></li>
                    <li><a href="#">Blog</a></li>
                    <li><a href="#">Support</a></li>
                    <li><a href="#">Contact Us</a></li>
                </ul>
               </div>
           
           

           
       </div>
        
    </header>
    
    
    <section id="s_first">
       <div class="container">
        
        <div class="main_title" >
        <p>Royal Season</p>
        
        </div>
        
           
         <div class="login-page clearfix">
         
  <div class="form">
    
    <p>If you are our regular customer you can log in to improve the quality of service</p>
    
    
    <form class="login-form">
      <input id="a" type="text" placeholder="Login"/>
      <input id="b" type="password" placeholder="Password"/>
      <button onclick="check()">login</button>
      
      <span>(Login: Royal; <br>
      Password: 4321)</span> 
      
    </form>
    
    
  </div>
  
  
</div>        
                
                 
                       
                             
                                   
                                               
       </div>
    
        
    </section>
    
    
    
    <section id="s_second">
        <div class="container">
            
           
           
           
           
           
            <div class="customiz">
             
             
             
            <div class="tuning">
               <h2 class="title">
                 Choice your room
             </h2>
                
               <div class="character ">
    
    <p class="item">Type of number  </p>
    <p class="item">Guests  </p>
    <p class="item">Number of nights  </p>
    <p class="item">View </p>
    <p class="item">Special wishes  </p>   
    
    
        
         </div>

         <div class="value">

             <select class="item" name="" id="room_type">
                 <option>Superior room </option>
                 <option>Suite room</option>
                 <option>Junior suite room </option>
                 <option>Superior room </option>
                 <option>Standart </option>
             </select>


             
             <p class="item quantity" >
                <span id="guests">___</span>
                 
                 <button class="plus_button" onclick="moreGuests()"> + </button>

                 <button class="plus_button" onclick="lessGuests()"> – </button></p>
                 
                 
             <p class="item quantity" > <span id="nights">___</span>
             
             
             
             
             <button class="plus_button" onclick="moreNights()"> + </button>

            <button class="plus_button" onclick="lessNights()"> – </button>
             
             </p>
             <select class="item " name="" id="with_view_on">
                 <option>on the Sea</option>
                 <option>on the Beach </option>
                 <option>on the Frest </option>
                 <option>do not metter </option>

             </select>
            <input class="item" id="wishes" type="text"> 
  
   
    
    
    
    
          
               </div> 
                
              
                  <button id="pick"> Pick a room</button>
     
                                            
            </div>    
             
             
             
             
             
         </div>
           
             
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
           
        </div>
        
        
        
        
    </section>
    
    
    
    
    
    
    
    <script src="Script.js">
    </script>
    
</body>
</html>