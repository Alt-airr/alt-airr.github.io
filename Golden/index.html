<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>Golden</title>
    <link rel="stylesheet" href="css/GoldenStyle.css">
</head>
<body>
   
    <header>
    <div class="container">
    
    <div class="heading clearfix">
    <img class="logo" src="img/logo.png" alt="logo">
    
    <nav>
    <ul class="menu">
        <li><a href="#">Home</a></li>
        <li><a href="#">Services</a></li>
        <li><a href="#">Portfolio</a></li>
        <li><a href="#">About</a></li>
        <li><a href="#">Contact</a></li>
    </ul>
    </nav>
    </div>
    
    
   <div class="titles">
      <div class="titles_first">
          Welcome to Our studio!
      </div>
       <h1>
           It's nice to Meet you
       </h1>
   </div>
       
        <a class="button" href="#"> Tell me more</a>
    
    </div>
    </header>
    
    <section>
        <div class="container">
          <div class="block">
          
           <div class="title">
               <h2>Services</h2>
          
            <p> Proin iaculis purus consequat sem cure </p>
            </div>
            
            <div class="services clearfix">
                
                <div class="service_item">
                    <img src="img/icon1.png" alt="icon1">
                    <h3>
                        E-Commerce
                    </h3>
                    <p class="info">
                        Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt.
                    </p>
                </div>
                
                <div class="service_item">
                    <img src="img/icon2.png" alt="icon2">
                    <h3>
                        Responsive Web
                    </h3>
                    <p class="info">
                       Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt.
                    </p>
                </div>
                
                <div class="service_item">
                    <img src="img/icon3.png" alt="icon3">
                    <h3>
                        Responsive Web
                    </h3>
                    <p class="info">
                       Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt.
                    </p>
                </div>
            </div>
            
        </div>
        </div>
    </section>
    
    
    
    <section id="portfolio">
        <div class="container">
             <div class="title">
               <h2>Our portfolio</h2>
            <p> Proin iaculis purus consequat sem cure </p>
            </div>
            
            <div class="works clearfix">
                <img  src="img/img1.jpg" alt="">
                <img  src="img/img2.jpg" alt="">
                <img  src="img/img3.jpg"s alt="">
            </div>
        </div>
    </section>
    
    <section #about>
        <div class="container">
            <div class="title">
               <h2>About Us</h2>
            <p> Proin iaculis purus consequat sem cure </p>
            </div>
            
            <div class="About_All">
            <div class="about">
             <div class="about_text1 clearfix">
                <h4>
                    July 2010
                    
                </h4>
                
                <h5>Our Humble Beginnings</h5>
                
                <p>
                  Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt. Proin 
iaculis purus consequat sem cure digni 
ssim. Donec porttitora entum suscipit.  
                </p>
                </div>
                
          <img class="about_img1" src="img/about1.jpg" alt="">
        </div>
           
            
            <div class="about">
             <div class="about_text2 clearfix">
                <h4>
                    January 2011 
                </h4>
                
                <h5>Facing Startup Battles</h5>
                
                <p>
                  Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt. Proin 
iaculis purus consequat sem cure digni 
ssim. Donec porttitora entum suscipit aenean 
rhoncus posuere odio in tincidunt. 
                </p>
                </div>
                
          <img class="about_img2" src="img/about2.jpg" alt="">
        </div>
           
           
               <div class="about">
             <div class="about_text1 clearfix">
                <h4>
                  December 2012 
                </h4>
                
                <h5>Enter The Dark Days</h5>
                
                <p>
                  Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt. Proin 
iaculis purus consequat sem cure digni. 
                </p>
                </div>
                
          <img class="about_img1" src="img/about3.jpg" alt="">
        </div>
            
            
         <div class="about">
             <div class="about_text2 clearfix">
                <h4>
                 February 2014 
                </h4>
                <h5>Our Triumph</h5>
                <p>
                  Proin iaculis purus consequat sem cure 
digni ssim. Donec porttitora entum suscipit 
aenean rhoncus posuere odio in tincidunt. Proin 
iaculis purus consequat sem cure digni 
ssim. Donec porttitora entum suscipit aenean. 
                </p>
                </div>
                
          <img class="about_img2" src="img/about4.jpg" alt="">
        </div>
            </div>
            
          </div> 
    </section> 
    
    <footer>
        <p>
        Â© Copyright 2014 FreebiesXpress.com
        
        
        </p>
    </footer>  
    
</body>
</html>